Dockerfile

```diff
@@ -25,6 +25,10 @@ ADD ./docker/install_python.sh /install_python.sh
 RUN chmod 755 /install_python.sh
 RUN /install_python.sh
 
+# Add the worker runner
+ADD ./docker/run_worker.sh /run_worker.sh
+RUN chmod 755 /run_worker.sh
```

Comment:
🙋‍♂ If we change the permissions for the file in git, we do not need to `chmod` it here.
(Same for the `install_python.sh` above)